games/entity/IDisplayableEntity.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+** EPITECH PROJECT, 2024
+** arcade-shared
+** File description:
+** IEntity
```

Comment:
This file doesn't name `IEntity` but `IDisplayableEntity`
```suggestion
** IDisplayableEntity
```